tools/internal_ci/linux/grpc_aws_experiment.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash -e
+
+# This script is responsible for remotely running tests on an ARM instance.
+# It should return a status code useful to the kokoro infrastructure.
+
+if [ -z "$KOKORO_KEYSTORE_DIR" ]; then
+    echo "KOKORO_KEYSTORE_DIR is unset. This must be run from kokoro"
+    exit 1
+fi
+
+AWS_CREDENTIALS=${KOKORO_KEYSTORE_DIR}/73836_grpc_aws_ec2_credentials
+
+## Setup aws cli
+curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
+unzip awscliv2.zip
```

Comment:
nit: the unzip command seems very verbose.

```
Archive:  awscliv2.zip
   creating: aws/
   creating: aws/dist/
  inflating: aws/THIRD_PARTY_LICENSES  
  inflating: aws/install             
  inflating: aws/README.md           
   creating: aws/dist/_struct/
   creating: aws/dist/awscli/
   creating: aws/dist/botocore/
   creating: aws/dist/cryptography/
   creating: aws/dist/cryptography-3.3.2-py3.8.egg-info/
   creating: aws/dist/docutils/
   creating: aws/dist/include/
   creating: aws/dist/lib/
   creating: aws/dist/zlib/
  inflating: aws/dist/_datetime.cpython-38-x86_64
  ...
  and many more files
```

let's use unzip -q or unzip -qq